src/store/actions.ts

```diff
@@ -80,7 +78,7 @@ export const setActiveComponent: any = ({ commit }, component): void => {
  * @param action the action to confirm
  * @returns True if confirmed, otherwise False
  */
-export const confirmAction: ActionIF = async ({ commit, getters }, action: string): Promise<boolean> => {
```

Comment:
`ActionIF` is wrong (even though the syntax allows it). The actual return type is `Promise<boolean>`.

Ditto the other changes below. It's just cleanup -- no functional impact.